Appy.GitDb.Core/Model/MergeInfo.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System.Collections.Generic;
+
+namespace Appy.GitDb.Core.Model
+{    
+    /// <summary>
+    /// The status of what happened as a result of a merge.
+    /// </summary>
+    public enum MergeStatus
+    {        
+        Succeeded = 0,
+        Conflicts = 1
+    }
+
+    public class MergeInfo
+    {
+        public string Message { get; set; }
+        public string SourceBranch { get; set; }
+        public string TargetBranch { get; set; }
+        public string CommitSha { get; set; }   
+        public MergeStatus Status { get; set; }        
+        public IList<ConflictInfo> Conflicts { get; set; }
+
+        public static MergeInfo NewSucceded(string sourceBranch, string targetBranch, string commitSha) =>
```

Comment:
Rename to `MergeInfo.Success` (allows the API to be more idiomatic) eg `return MergeInfo.Success(...)`